language-server/language-server-api/.js/src/main/scala/aqua/lsp/TypeJs.scala

```diff
@@ -0,0 +1,105 @@
+package aqua.lsp
+
+import aqua.types.*
+
+import scala.scalajs.js.Dictionary
+import scala.scalajs.js.JSConverters.*
+import scalajs.js
+
+sealed trait TypeJs extends js.Object {
```

Comment:
Why not model them as `enum`? 
Also I don't quite understand: don't you need to annotate class fields with `@JSExport`? Otherwise they should not be visible in js, should they?

Reply:
`enum` don't work with `js.Object` as far as I know. 
Also, `@JSExport` makes scala class accessible from JS. `extends js.Object` makes these objects AS JS objects. So, I think we will rewrite definitions on `js.Object` too, to avoid redundant proxy conversions